packages/contentstack-import/src/import/modules-js/labels.js

```diff
@@ -108,6 +108,15 @@ module.exports = class ImportLabels {
         })
         .catch(function (error) {
           // error while importing labels
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).label !== undefined) {
+            if (JSON.parse(error.request.data).label.name) {
+              message_content_type =
+                ' Due to label titled - ' + JSON.parse(error.request.data).label.name;
```

Comment:
@cs-raj Can you please validate the error messages with the docs team?

Reply:
Validated with Varsha and made the changes